doc/radosgw/admin.rst

```diff
@@ -473,6 +473,157 @@ commands. ::
    be restarted for the changes to take effect.
 
 
+Rate Limit Management
+=====================
+
+The Ceph Object Gateway enables you to set rate limits on users and buckets. 
+rate limit include the maximum number of read ops and write ops per minute
+and how many write bytes or read bytes per minute a user or bucket can make.
+Requests that are using GET or HEAD method in the REST request considered as read, otherwise it is write.
```

Comment:
```suggestion
Requests that are using GET or HEAD method in the REST request are considered as "read requests", otherwise they are considered as "write requests".
```